tasks/br/geo.py

```diff
@@ -200,18 +214,38 @@ def columns(self):
             weight=0,
             targets={geom: GEOM_REF},
         )
-        return OrderedDict([
-            ('geom_id', geom_id),   #
-            ('the_geom', geom),     # the_geom
-        ])
 
+        if self.resolution != 'setores_censitarios':
+            geom_name = OBSColumn(
+                id = self.resolution + '_name',
+                name="Name of the {}".format(GEOGRAPHY_NAMES[self.resolution]),
+                type='Text',
+                weight=1,
+                tags=[sections['br'], subsections['names']],
+                targets={geom: GEOM_NAME}
+            )
+            cols['{}'.format(GEOGRAPHY_PROPERNAMES[self.resolution])] = geom_name
+        else:
+            for region in GEOGRAPHY_PROPERNAMES[GEO_I]:
```

Comment:
Census tracts in Brazil (aka setores_censitarios) don't have proper names, but they are identified by a mesoregion and microregion. So this columnstask adds a column for each region.